### ISSUE TITLE
popper.min.js call a missing file inside understrap instead of  understrap-child

### ISSUE THREAD

**Issue by JDmnT** [understrap/understrap-child#195]
Hi;

It is not a huge issue but
at least my understrap-child seek for ./wp-content/themes/understrap/js/popper.min.js
and not ./wp-content/themes/understrap-child/js/popper.min.js

I resolved this by copying popper*.js into understrap/js/

but I also think it must be fix somewhere [here](https://github.com/understrap/understrap-child/search?q=popper.min.js&unscoped_q=popper.min.js)

my guest is is in [function](https://github.com/understrap/understrap-child/blob/master/functions.php#L24) but my wordpress dev skill are limited ;)

Cheer!

**Comment by Podtwal** [comment]
Hello. 
I can confirm the circumstance. My "solution" is, in the child function.php
-> get_template_directory_uri() . /js/popper.min.js', array(), false);
	wp_enqueue_script( 'popper-scripts', 
as follows...
-> **get_stylesheet_directory_uri()** . '/js/popper.min.js', array(), false);

There may be an inconsistency between the parent and child theme?
In the parent theme on Git there is a note in the directory "js" -> _we don´t need popper.js anymore because it is in bootstrap.bundle.js_

This may not be transferred to the child theme?

**Comment by jaredsnider** [comment]
Also curious about this, all my Understrap sites consistently throw the `Failed to load resource: the server responded with a status of 404 (Not Found) popper.min.js:1` error and was wondering if I had done something wrong or if its something that needs to be removed from the parent/child theme.

**Comment by sabwebsites** [comment]
I'm currently using the Understrap child theme in Wordpress. I can't get popovers, tooltips or vertical pills to work.

Here is a live example: [Sample Page][1]

Popper.js seems to be included with the child theme. Any suggestions? I'm not sure what is the missing puzzle.

I'm using understrap documentation from GitHub and using [parent][2] and understrap [child theme][3] files as they are.

The get_stylesheet_directory_uri is correct in my functions php and there are both popper.js + popper.min.js files in the child theme's /js directory but I still don't get it why isn't working. 

I did try a couple of things, like turning on and off all plugins or switching back to parent theme while I've moved the popper files back in the parent js folder, however, still no success...

  [1]: https://arondomains.com/sample-page/
  [2]: https://github.com/understrap/understrap
  [3]: https://github.com/understrap/understrap-child
